keys/cryptostore/holder.go

```diff
@@ -32,13 +39,35 @@ func (s Manager) assertKeyManager() keys.Manager {
 //
 // algo must be a supported go-crypto algorithm:
 //
-func (s Manager) Create(name, passphrase, algo string) (keys.Info, error) {
+func (s Manager) Create(name, passphrase, algo string) (keys.Info, string, error) {
 	gen, err := getGenerator(algo)
 	if err != nil {
-		return keys.Info{}, err
+		return keys.Info{}, "", err
 	}
 	key := gen.Generate()
 	err = s.es.Put(name, passphrase, key)
+	if err != nil {
+		return keys.Info{}, "", err
+	}
+	seed, err := s.codec.BytesToWords(key.Bytes())
+	phrase := strings.Join(seed, " ")
+	return info(name, key), phrase, err
+}
+
+func (s Manager) Recover(name, passphrase, seedphrase string) (keys.Info, error) {
```

Comment:
Add Lint comments throughout! All exposed functions should have a header comment.